### PR TITLE
Set log levels for logging statements

### PR DESCRIPTION
**Pull request by nein09**
I've made my best guess at what the various output levels should be, and some log statements that were commented out are now back in, as a `log.Trace`.

This is structured so that you can pass Gleaner a `-log` flag, with a level. The default is `warn`, which makes my command-line run look like this:

```
confusion~/gleaner/cmd/gleaner(mm-dev--log-levels|…) % ./gleaner_darwin -cfg config.yaml 
EarthCube Gleaner
 100% |██████████████████████████████████████| (273/273, 4 it/min)             
 100% |███████████████████████████████████████████| (375/375, 105 it/s)        
 100% |████████████████████████████████████| (20438/20438, 147 it/s)           
 100% |████████████████████████████████████████| (2908/2908, 163 it/s)         
 100% |███████████████████████████████████████████| (529/529, 133 it/s)        
 100% |████████████████████████████████████████| (7293/7293, 134 it/s)         
 100% |███████████████████████████████████████████| (912/912, 103 it/s)        
 100% |███████████████████████████████████████████| (709/709, 64 it/s)         
Calling cleanUp on a successful run
On success, will, if flagged, copy the log file to object store and delete it
```
`trace` is the most verbose.

### CODE REVIEW

File: pkg/cli/root.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ func Execute() {
 }
 
 func init() {
-	log.Println("EarthCube Gleaner")
+	log.Info("EarthCube Gleaner")
```
Comment (valentinedwv) [inline pkg/cli/root.go:52]: Will need to update this file to enable the logging for the glcon program.

**Review by valentinedwv — APPROVED** [review]
looks good. I'll need to update the glcon command to enable logging,

**Review by fils — APPROVED** [review]
thanks!   we can worry about adjusting levels as an ongoing thing...   great to have the first pass done though!

**Comment by fils** [comment]
@nein09    Melinda, sorry it took so long for me to get to this.    I see this pull request has some conflicts above.   I wonder if I accidentally did the pull requests in the wrong order and caused this.      

If it's something you can resolve feel free to resolve it and merge in.   Otherwise maybe we can set up a time to go over this next week.   

Would enjoy a coding meeting with you anyway next week.  I need to update you on a few some developments.   We can arrange something via slack.

Thanks
Doug

**Comment by nein09** [comment]
@fils no worries, I can take care of the conflicts - but in resolving them, I noticed a whoooole bunch of logging statements that I'd missed before.

I'll get that in as soon as I can.

**Comment by fils** [comment]
@nein09 sounds good..   just let me know.  I have some updates in a branch I want to try and merge in..   I've been lax so trying to get back on top of some code things..     would be good to have another coding meeting sometime if it works for everyone

**Comment by nein09** [comment]
@fils Another meeting would be good, I agree. I just tested this out and I think it's good to go, so I'm going to merge it now.

**Pull request merged**